System/battery_estimator.5s.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+# <xbar.title>Battery Estimator</xbar.title>
+# <xbar.version>v1.0</xbar.version>
+# <xbar.author>Lakeland</xbar.author>
+# <xbar.author.github>Lakeland97</xbar.author.github>
+# <xbar.desc>Estimates when the battery will run out based on remaining time.</xbar.desc>
+# <xbar.dependencies>bash</xbar.dependencies>
```

Comment:
please add an image with the `<xbar.image>` tag.

Reply:
@matryer 

I've added an image to the plugin. Still needs a `chmod +x`, but I'll merge this in and do that in a separate PR.